content/ar/contribute/_index.md

```diff
@@ -0,0 +1,141 @@
+
+---
+
+title: كيف أساهم
+
+toc_hide: true
+
+menu:
+
+main:
+
+weight: 10
+
+pre: <i class='fas fa-pen-square'></i>
+
+---
```

Comment:
```suggestion
---
title: كيف أساهم
toc_hide: true
menu:
  main:
    weight: 10
    pre: <i class='fas fa-pen-square'></i>
---
```
@same7ammar I cannot find the menu for `How to contribute` in the preview (https://deploy-preview-406--cncfglossary.netlify.app/ar/). I guess it is because of a wrong format of header.

Reply:
@seokho-son yeah correct I changed it be suggested but the menu is still not showing .  Please suggest if there is any config missing .

```
---
title: كيف أساهم
toc_hide: true
menu:
    main:
    weight: 10
    pre: <i class='fas fa-pen-square'></i>
---
```

Reply:
@same7ammar Your correction seems not applied. I cannot find a new commit. :)

Reply:
@seokho-son my bad :) , it should be working fine now. Could you please take a look .

Reply:
> @same7ammar could you add `style-guide/_index.md` the to this PR? to fulfill [3-4. Minimum required content](https://github.com/cncf/glossary/blob/main/LOCALIZATION.md#3-4-minimum-required-content) in one PR.
> 
> Home: content/en/_index.md -> content/ar/_index.md How to contribute: content/en/contribute/_index.md -> content/ar/contribute/_index.md Style guide: content/en/style-guide/_index.md -> content/ar/style-guide/_index.md

@seokho-son  Style guide has been added now .